test/common/pixel-tests

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+set -eux
+
+SUBDIR=test/reference
+
+GITHUB_BASE="${GITHUB_BASE:-cockpit-project/cockpit}"
+GITHUB_REPOSITORY="${GITHUB_BASE%/*}/pixel-test-reference"
+CLONE_REMOTE="https://github.com/${GITHUB_REPOSITORY}"
+PUSH_REMOTE="git@github.com:${GITHUB_REPOSITORY}"
```

Comment:
If you intend to use this from workflows, you'll have to push through https, with ~/.git-credentials. See e.g. po-refresh or publish-dist workflows how to set that up.

Reply:
This came back, so unresolving. This is *not* a blocker for landing this PR (as that is a bit urgent), but a workflow must be able to override this variable.

Reply:
Yes, this very likely needs more changes.  $CLONE_REMOTE is only used to write the .gitmodules file, and $PUSH_REMOTE is used regardless of what is in .gitmodules...  Maybe we should just always push via https...